packages/uikit/src/hooks/browser/useRecommendations.ts

```diff
@@ -0,0 +1,41 @@
+import { useUserCountry } from '../../state/country';
+import { useQuery } from '@tanstack/react-query';
+import { QueryKey } from '../../libs/queryKey';
+
+export interface CarouselApp extends PromotedApp {
+    poster: string;
+}
+
+export interface PromotedApp {
+    name: string;
+    description: string;
+    icon: string;
+    url: string;
+    textColor?: string;
+}
+
+export interface PromotionCategory {
+    id: string;
+    title: string;
+    apps: PromotedApp[];
+}
+
+export interface Recommendations {
+    categories: PromotionCategory[];
+    apps: CarouselApp[];
+}
+
+export function useRecommendations() {
+    const country = useUserCountry();
+    const lang = country.data || 'en';
+
+    return useQuery<Recommendations, Error>([QueryKey.featuredRecommendations, lang], async () => {
+        const result = await (
+            await fetch(`https://api.tonkeeper.com/apps/popular?lang=${lang}`)
```

Comment:
Could you please move method and interfaces to tonendpoint  class?

the same as `/fiat/methods`
https://github.com/tonkeeper/tonkeeper-web/blob/d87589789038aae1e96bb00a32de78f0879a6919/packages/core/src/tonkeeperApi/tonendpoint.ts#L171


```js
    const { data: country } = useUserCountry();
    const { tonendpoint } = useAppContext();
    
    const data = await getBrowserApps(tonendpoint, country)
```

`tonendpoint class` will automatically add language, country, build number to query params